Chapter-1-initial-architecture/Src/Fitnet.IntegrationTests/Contracts/PrepareContract/PrepareContractTests.cs

```diff
@@ -6,16 +6,14 @@ namespace EvolutionaryArchitecture.Fitnet.IntegrationTests.Contracts.PrepareCont
 using Common.TestEngine.Configuration;
 using EvolutionaryArchitecture.Fitnet.Common.ErrorHandling;
 
-public sealed class PrepareContractTests : IClassFixture<WebApplicationFactory<Program>>,
+public sealed class PrepareContractTests(
+    WebApplicationFactory<Program> applicationInMemoryFactory,
+    DatabaseContainer database) : IClassFixture<WebApplicationFactory<Program>>,
     IClassFixture<DatabaseContainer>
 {
-    private readonly HttpClient _applicationHttpClient;
-
-    public PrepareContractTests(WebApplicationFactory<Program> applicationInMemoryFactory,
-        DatabaseContainer database) =>
-        _applicationHttpClient = applicationInMemoryFactory
-            .WithContainerDatabaseConfigured(database.ConnectionString!)
-            .CreateClient();
+    private readonly HttpClient _applicationHttpClient = applicationInMemoryFactory
```

Comment:
I did it a bit different in chapter 2 but will follow with your approach (will adjust it in separate branch)